web/app/js/components/Deployment.jsx

```diff
@@ -188,12 +194,28 @@ export default class Deployment extends React.Component {
     );
   }
 
+  renderDeploymentTitle() {
+    return (
+      <div className="deployment-title">
+        <h1>{this.state.deploy}</h1>
+        {
+          !this.state.added ? (
+            <div className="unadded-message">
+              <div className="status-badge unadded"><p>UNADDED</p></div>
+              <div className={"incomplete-mesh-message"}>
+                <div className="instructions emphasis">{this.state.deploy} has not been added to the service mesh</div>
+                <div className="instructions">Add the deployment to the deployment.yml file</div>
+                <div className="instructions">Then run <code>kubectl inject deployment.yml | kubectl apply -f - </code> to add the deployment to the service mesh</div>
+              </div>
+            </div>
+          ) : null
+        }
+      </div>
+    );
+  }
+
   renderContent() {
```

Comment:
there is no need to have a renderContent function if all it does is call renderSections